docs/visualize/timelion.asciidoc

```diff
@@ -504,3 +504,52 @@ image::images/timelion-conditional04.png[]
 {nbsp}
 
 For additional information on Timelion conditional capabilities, go to https://www.elastic.co/blog/timeseries-if-then-else-with-timelion[I have but one .condition()].
+
+[float]
+[[timelion-deprecation]]
+=== Timelion App deprecation
+
+In *8.0* and later, the Timelion app - deprecated since 7.0 - is going to be removed. There are actions you need to complete to copy your existing Timelion worksheets to a dashboard.  
+
+NOTE: Only the Timelion app is deprecated, not Timelion as a visualization (on dashboards and in the visualize application) or Timelion in Canvas. 
+
+[role="screenshot"]
+image::images/timelion-app.png[]
+{nbsp}
+
+[float]
+[[timelion-app-to-vis]]
+==== Create a dashboard from a Timelion worksheet
+
+Timelion worksheets are deprecated, but they can easily be replaced by a dashboard. The process is the same as adding a new visualization to a dashboard but the Timelion charts should also be migrated to **Visualize**. 
```

Comment:
To replace a Timelion worksheet with a dashboard, follow the same process for adding a visualization.  In addition, you must migrate the Timelion worksheet to Visualize.